conf/v2_route/task.api

```diff
@@ -537,6 +537,43 @@ GET     /tasks/random                               @org.maproulette.controllers
 PUT     /tasks/box/:left/:bottom/:right/:top        @org.maproulette.framework.controller.TaskController.getTasksInBoundingBox(left:Double, bottom:Double, right:Double, top:Double, limit:Int ?= 10000, page:Int ?= 0, excludeLocked:Boolean ?= false, sort:String ?= "", order:String ?= "ASC", includeTotal:Boolean ?= false, includeGeometries:Boolean ?=false, includeTags:Boolean ?= false)
 ###
 # tags: [ Task ]
+# summary: Retrieves Task Marker Data within a bounding box
+# description: Retrieves task marker data within a given bounding box.
+# responses:
+#   '200':
+#     description: The list of markers representing Tasks that match the search criteria within the bounding box.
+#     content:
+#       application/json:
+#         schema:
+#           type: array
+#           items:
+#             $ref: '#/components/schemas/org.maproulette.framework.model.ClusteredPoint'
+# parameters:
+#   - name: left
+#     in: path
+#     description: The minimum latitude for the bounding box.
+#   - name: bottom
+#     in: path
+#     description: The minimum longitude for the bounding box.
+#   - name: right
+#     in: path
+#     description: The maximum latitude for the bounding box.
+#   - name: top
+#     in: path
+#     description: The maximum longitude for the bounding box.
```

Comment:
Nit: these descriptions are mixed up; `left` and `right` are the min and max _longitude_ and `bottom` and `top` are the min and max _latitude_.

This text was copy-pasted from the existing `/tasks/box/:left/:bottom/:right/:top` endpoint and should be fixed there too if you don't mind 🙂